lib/taskana-core/src/main/java/pro/taskana/task/api/TaskQuery.java

```diff
@@ -202,6 +202,14 @@
    */
   TaskQuery ownerIn(String... owners);
 
+  /**
+   * Filter out owners.
```

Comment:
In the other java doc for the api we usually also added "This is an exact match". Also we always added "of the task". Could you change this to "Filter out owners of the task. This is an exact match."?